packages/components/src/layouts/Navigation/Tab/TabBar/DesktopTabItem.tsx

```
@@ -36,8 +38,26 @@
     showAvatar = false,
     ...rest
   } = props;
+
+  const stackRef = useRef<TamaguiElement>(null);
+  const openActionList = useRef<() => void | undefined>();
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)